plugins/catalog-unprocessed-entities/src/components/FailedEntities.tsx

```diff
@@ -93,14 +93,18 @@ const RenderErrorContext = ({
  * easily understand the times.
  */
 const convertTimeToLocalTimezone = (strDateTime: string | Date) => {
-  const dateTime = DateTime.fromFormat(
+  let dateTime = DateTime.fromFormat(
```

Comment:
This code confuses me (what was there before).

Why would we ever stringify _in a local unknown format and then try to parse it again_ when the input was already a Date that has perfect strict semantics already? :D

This should all be changed around so that IF the type is string, then try to parse the two well known formats (the one that was here before `yyyy-MM-dd hh:mm:ss`, as well as ISO, no `toLocaleString` necessary at all. But if it's NOT string, then it's a Date, so use `DateTime.fromJSDate(strDateTime)` on it instead.

Reply:
This function should also have some tests added to show what is being covered.